physionet-django/user/forms.py

```diff
@@ -668,8 +668,6 @@ def __init__(self, user, *args, **kwargs):
         self.fields['training_type'].initial = self.training_type
 
         if self.training_type is not None:
-            self.fields['training_type'].help_text = self.training_type.description
```

Comment:
Does this need to be removed? In https://github.com/MIT-LCP/physionet-build/pull/1529/commits/2938c10470a414ab3dfd6bd20b91170d01e27323#diff-1616a969a01973c697c08fe3fe6011689bdaa0d7684d5746f130651bc7572667 it looks like @adrianryt is fixing a problem with the rendering of the field in the template. Are these two changes addressing the same issue in different ways?

Reply:
@tompollard Seems Rafał removed it in one of his commits. I restored it to the way it's done on dev.